e2e-pw/src/oss/poms/modal/leva.ts

```diff
@@ -0,0 +1,36 @@
+import { Locator, Page, expect } from "src/oss/fixtures";
+import { ModalPom } from ".";
+
+export class ModalLevaPom {
+  readonly page: Page;
+  readonly locator: Locator;
+  readonly assert: LevaAsserter;
+
+  constructor(page: Page, private readonly modal: ModalPom) {
```

Comment:
The `Page` is only used by the constructor for creating a `Locator`. Maybe a locator of the `ModalPom` can be used to find `"#fo-leva-container"`? Also, the `dataCy` plus `getByTestId` pattern is preferred over an HTML `#id` e.g. [here](https://github.com/voxel51/fiftyone/blob/59e6e6e1973edb7da246018e111c44267dfe9591/e2e-pw/src/oss/poms/selector.ts#L17)

@sashankaryal is the e2e guru, but this has been my observation :)